data.json

```diff
@@ -27,6 +27,31 @@
 		"url_regex": "https%3A%2F%2Fwww.porntrex.com%2Fvideo%2F.*%2F.*",
 		"video_xpath": "//div[@id='kt_player']/div[contains(@class, 'player')]/video"
 	},
+	{
+		"url_regex": "https%3A%2F%2Fwww.povaddict.com%2F.*",
+		"video_xpath": "//video[@id='video-player_html5_api']"
+	},
+	{
+		"url_regex": "https%3A%2F%2Fwww.manyvids.com%2FVideo%2F.*",
+		"video_xpath": "//video[@class='rmp-object-fit-contain rmp-video']"
+	},
+	{
+		"url_regex": "https%3A%2F%2Fwww.mega.nz%2F.*",
+		"video_xpath": "//video[@id='video']"
+	},

```

Comment:
Matches the entire site, should only match videos
Also mega doesn't always have www infront
https://.*mega.nz/file/.*
'https%3A%2F%2F.*mega.nz%2Ffile%2F.*'